### ISSUE TITLE
Song select BGM & song preview play simultaneously when entering/returning to song select

### ISSUE THREAD

**Issue by twopointzero** [AioiLight/TJAPlayer3#140]
Reporting this quickly on behalf of a discord user, so please get in touch if you need more detail.

It would seem that SimpleStyle should have song select BGM added to it so we can spot issues like this in the future. Apparently, when entering song select from the main menu, or when returning to song select from song play, both the song select BGM and the song preview will play simultaneously until you scroll to a different song.

**Comment by AioiLight** [comment]
曲の生成が少しでも遅れれば、この問題は起こらないようです。
SimpleStyleにもBGMを追加したいですが、フリーBGMを探す必要(再配布OK)があります。

If the generation of music is delayed even a little, this problem does not seem to happen.
I'd like to add BGM to SimpleStyle, but there is a need to look for free BGM (redistribution OK).

**Comment by twopointzero** [comment]
Yes, a tiny delay would likely be enough to cause the right code to run in the right order so that the song select BGM is either never started or is faded down just barely after it starts fading up, while it is still too quiet to hear.
